### ISSUE TITLE
asr model missing from bundle

### ISSUE THREAD

**Issue by adam-zethraeus** [LaurentMazare/moshi-swift#3]
Thanks for sharing this — I'm enjoying prodding around :)

FYI: When I run the macOS app's asr model or the or cli `asr-file` I see that `asr-300m-f28fe6d5@450.safetensors is missing` — and I don't think it's set up to be downloadable from hf.

**Comment by LaurentMazare** [comment]
Right, this is an internal model at the moment, I have to find a good way to make these only displayed when they are actually available.

**Comment by LaurentMazare** [comment]
We've finally released some ASR models and this codebase should be compatible with them now. The 1b model can be found on [kyutai/stt-1b-en_fr-mlx](https://huggingface.co/kyutai/stt-1b-en_fr-mlx).

**Comment by LaurentMazare** [comment]
Closing as the ASR is now available, feel free to re-open if needed.
